### ISSUE TITLE
Support use of plantuml syntax 

### ISSUE THREAD

**Issue by maihacke** [devonfw/devonfw-guide#70]
[Asciidoctor diagram](https://asciidoctor.org/docs/asciidoctor-diagram/) supports different inline diagramm types which are very useful for our guide. For some types additonal setup for the generation process is required.
Supporting plantuml for creating UML diagrams directly in the devonfw docs would be very useful. For example we use plantuml in the [Kafka module](https://github.com/devonfw/devon4j/blob/develop/documentation/guide-kafka.asciidoc). Currently it required manual steps do generate the SVGs and PNGs to include them.

**Comment by maybeec** [comment]
I have seen, that there is already an example here: https://github.com/asciidoctor/asciidoctor-maven-examples/tree/master/asciidoctor-diagram-example
So we should investigate to adapt the process respectively for PDF and Website generation.

**Comment by maihacke** [comment]
Just to mention it here. https://www.diagrams.net/ could also be a good alternative for more complexes. Maybe we could found somebody to evaluate alternatives and make an proposal. Also supporting multiple alternatives could be feasible...

**Comment by maybeec** [comment]
yes but please take in mind the compatibility with our docs build to render PDFs and the website automatically.

**Comment by maihacke** [comment]
Yes, sure. But if I look at diagrams.net there are multiple embedding and exporting options, so this should not be a show stopper.
